app/views/users/index.html.erb

```diff
@@ -1 +1,22 @@
-<h2>HELLO RAILS index</h2>
\ No newline at end of file
+<div class='container'>
+  <h1 class= 'page_title'>List of Users</h1>
+  <div>
+      <ul>
+    <% @users.each do |user| %>
+    <% @posts = Post.where(author_id: user.id) %>
+        <li>
+          <a href= "/users/<%= user.id %>" class = 'user_clickable'>
+            <div class = 'user_container'>
+              <img class = 'user_profile' src= '<%= user.photo %>' alt='photo comes here' width="100" height="100" />
+              <div class = 'info_holder' >
+                <h3><%= user.name %></h3>
+                <p class = 'post_counter'>Post :  <%= @posts.length %></p>
+              </div>
+            </div>
+          </a> 
+        </li>
+    <% end %>
+      </ul>
```

Comment:
[OPTIONAL]
- [ ] You can consider the option below as a better practice instead of using `@objects.each`. You could create `app/views/users/_user.html.erb`, with this sample code. For reference, [look here](https://guides.rubyonrails.org/layouts_and_rendering.html)

```ruby
    # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```